modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ActivationStatus/DXPCloud/index.js

```diff
@@ -43,10 +44,23 @@ const SetupDXPCloudModal = ({
 }) => {
 	const [formAlreadySubmitted, setFormAlreadySubmitted] = useState(false);
 
+	const submittedModalTexts = {
+		paragraph:
+			'Return to the product activation page to view the current Activation Status',
+		subtitle: `We'll need a few details to finish building your DXP
+		environment(s).`,
+		text:
+			'Another user already submitted the DXP Cloud activation request.',
+		title: 'Set up DXP Cloud',
+	};
+
```

Comment:
Hi @igordmf, constant values should be good created before of component as not depend the component render

Reply:
Done! Thank you!